### PR TITLE
docs: Fix messed up accessibilityRole and accessibilityStates

### PR DESCRIPTION
**Pull request by haruelrovix**
In this PR:

- Run sync-guides
- Fix unavailable `accessibilityRole` and `accessibilityStates` for RN older version (< v0.60)

### CODE REVIEW

**Comment by haruelrovix** [comment]
Refer to changes on React Native side:

- https://github.com/facebook/react-native/blob/v0.59.10/Libraries/Components/View/ViewAccessibility.js#L43
- https://github.com/facebook/react-native/blob/v0.60.0/Libraries/Components/View/ViewAccessibility.js#L16

cc @charpeni @hramos

File: website/versioned_docs/version-0.5/native-components-ios.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Let's say we want to add an interactive Map to our app - might as well use [`MKM
 
 Native views are created and manipulated by subclasses of `RCTViewManager`. These subclasses are similar in function to view controllers, but are essentially singletons - only one instance of each is created by the bridge. They expose native views to the `RCTUIManager`, which delegates back to them to set and update the properties of the views as necessary. The `RCTViewManager`s are also typically the delegates for the views, sending events back to JavaScript via the bridge.
 
-To expose a view you can:
+To exposing a view you can:
```
Comment (hramos) [inline website/versioned_docs/version-0.5/native-components-ios.md]: I don't think this one needs to be changed.

File: website/versionedDocsBlacklist.json [hunk 2]
```diff
@@ -17,7 +17,6 @@
   "navigation",
   "images",
   "animations",
-  "accessibility",
```
Comment (hramos) [inline website/versionedDocsBlacklist.json]: Leave "Accessibility" here, as it's an unversioned Guide doc.

**Comment by rachelnabors** [comment]
Since nothing has been done on this to address the review request in several months, I'm going to close the PR for now. Author is welcome to reopen it when they can participate again!

**Pull request closed without merging**